tools/monitoring/yaml-handler/parser.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The yamlHandler is responsible for fetching, parsing config yaml file. It also allows user to
+// retrieve a particular record from the yaml.
+
+package yamlHandler
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+)
+
+type AlertConditions struct {
```

Comment:
```suggestion
type AlertCondition struct {
```

Reply:
this struct is a collection of conditions instead of one single condition